src/main/java/frc/robot/subsystems/TankSubsystem.java

```diff
@@ -20,37 +23,57 @@ public class TankSubsystem extends SubsystemBase {
   private final WPI_TalonSRX rightMotor;
   private final WPI_TalonSRX rightMotorFollower;
 
+  private final XboxController controller;
+
+  private double leftSpeed;
+  private double rightSpeed;
+  
   /** Creates a new TankSubsystem. */
   public TankSubsystem() {
     /* Left Motors */
     leftMotor = new WPI_TalonSRX(LEFT_FRONT_MOTOR_ID);
     leftMotorFollower = new WPI_TalonSRX(LEFT_BACK_MOTOR_ID);
     leftMotor.setNeutralMode(NeutralMode.Brake);
+    leftMotorFollower.setNeutralMode(NeutralMode.Brake);
     leftMotorFollower.follow(leftMotor);
 
     /* Right Motors */
     rightMotor = new WPI_TalonSRX(RIGHT_FRONT_MOTOR_ID);
     rightMotorFollower = new WPI_TalonSRX(RIGHT_BACK_MOTOR_ID);
     rightMotor.setNeutralMode(NeutralMode.Brake);
+    rightMotorFollower.setNeutralMode(NeutralMode.Brake);
     rightMotor.setInverted(InvertType.InvertMotorOutput);
     rightMotorFollower.setInverted(InvertType.FollowMaster);
     rightMotorFollower.follow(rightMotor);
+
+    /* Controls */
+    controller = new XboxController(OperatorConstants.kDriverControllerPort);
+
+    /* Default Speeds upon Initialization */
+    leftSpeed = 0.0;
+    rightSpeed = 0.0;
   }
 
   @Override
   public void periodic() {
-    setMotors(0.5, 0.5);
+    double throttlePower = -controller.getLeftY() * THROTTLE_SCALE;
+    double turnPower = controller.getRightX() * TURN_SCALE;
```

Comment:
One consequence of having different throttle and turn scales is that pushing fully forward and to one side no longer turns around the side of the robot. We can keep it like this for now, but we might want to change it later depending on driver preferences.